AIPs/aip-5.md

```diff
@@ -1,212 +1,176 @@
 
-The original Ampleforth linear rebasing function was intended to make minimal assumptions about how price reacts to expansion and contraction (i.e. expansion would slow price increases by translating them into supply increases and contraction would slow price decreases by translating them into supply decreases). Said differently, the main purpose of the function is to point supply changes in the right direction without trying to predict the exact magnitude of supply change needed for the fastest reversion to the price target.
+The original Ampleforth linear rebase function was intended to make minimal assumptions about how price reacts to expansion and contraction--i.e. expansion would slow price increases by translating them into supply increases and contraction would slow price decreases by translating them into supply decreases. Hence the main purpose of the function is to point supply change in the right direction without trying to predict the magnitude of supply change needed for the fastest convergence to the price target.
 
-Now that we have observed a year’s worth of market history and analyzed the data, we can see that extreme market scenarios can have outsized effects on AMPL supply, which then require large and prolonged supply corrections. The goal of this change is to:
+Now that we have observed a year’s worth of market history and analyzed the data, we can see that extreme market scenarios can have outsized effects on AMPL supply, which then require prolonged supply corrections. The goal of this change is to:
 
-- Limit protocol sensitivity to short-lived, but extreme market conditions that can wildly expand or contract supply. Currently, the protocol already has a maximum contraction percentage, but a potentially unlimited expansion percentage.
 
-- Create symmetry between price signals to expand supply and signals to contract supply—e.g. a signal to double supply (price of $2), should be perfectly offset by a signal to halve supply (price of $0.5). This is currently not the case, as explained below.
 
-These changes are expected to make AMPL more robust as a base money in decentralized finance applications. AMPL will keep its dynamic properties to change supply to match demand. In addition AMPL will be enhanced by protections against short-lived but extreme market conditions. The protocol is also expected to remain closer to its target, now that larger supply adjustments will happen closer to the target price.
+*   Limit protocol sensitivity to short-lived, but extreme market conditions that can disproportionately change supply.
 
-### Motivation for Symmetry
+Additional side benefits from having a cap on daily supply changes:
```

Comment:
Suggest "Additional side benefits:" instead of "Additional side benefits from having a cap on daily supply changes:"